scripts/components/OpenSearch-Dashboards/build.sh

```diff
@@ -109,6 +109,10 @@ case $PLATFORM-$DISTRIBUTION-$ARCHITECTURE in
         ;;
 esac
 
+echo "Setting node version"
+source $NVM_DIR/nvm.sh
+nvm use
```

Comment:
Add a check if $NVM_DIR isn't defined then send them to the setup guide - this would be a good fast follow

Reply:
Oh yes that would actually be a really clean solution that make me feel better about this code being in here. I've created an issue https://github.com/opensearch-project/opensearch-build/issues/1922 but trying my best not to be online so I might not be able to get this as a fast follow.